rest-tests/src/test/java/io/syndesis/qe/rest/tests/integrations/DropboxSteps.java

```diff
@@ -0,0 +1,86 @@
+package io.syndesis.qe.rest.tests.integrations;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import cucumber.api.PendingException;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import io.syndesis.common.model.action.Action;
+import io.syndesis.common.model.action.ConnectorDescriptor;
+import io.syndesis.common.model.connection.Connection;
+import io.syndesis.common.model.connection.Connector;
+import io.syndesis.common.model.integration.Step;
+import io.syndesis.common.model.integration.StepKind;
+import io.syndesis.qe.accounts.Account;
+import io.syndesis.qe.bdd.AbstractStep;
+import io.syndesis.qe.bdd.entities.StepDefinition;
+import io.syndesis.qe.bdd.storage.StepsStorage;
+import io.syndesis.qe.endpoints.ConnectionsEndpoint;
+import io.syndesis.qe.endpoints.ConnectorsEndpoint;
+import io.syndesis.qe.utils.RestConstants;
+import io.syndesis.qe.utils.S3BucketNameBuilder;
+import io.syndesis.qe.utils.TestUtils;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * June 7, 2018 Red Hat
+ *
+ * @author sveres@redhat.com
+ */
+@Slf4j
+public class DropboxSteps extends AbstractStep {
+
+    @Autowired
+    private StepsStorage steps;
+    @Autowired
+    private ConnectionsEndpoint connectionsEndpoint;
+    @Autowired
+    private ConnectorsEndpoint connectorsEndpoint;
+
+    public DropboxSteps() {
+    }
+
+    @And("^create Dropbox download START action step with file path: \"([^\"]*)\"$")
+    public void createDropboxDownloadAction(String filePath) {
+        final Connector dropboxConnector = connectorsEndpoint.get("dropbox");
+        final Connection dropboxConnection = connectionsEndpoint.get(RestConstants.DROPBOX_CONNECTION_ID);
+        final Action dropboxDownloadAction = TestUtils.findConnectorAction(dropboxConnector, "io.syndesis:dropbox-download-connector");
+        final Map<String, String> properties = TestUtils.map("remotePath", filePath);
+        final ConnectorDescriptor connectorDescriptor = getConnectorDescriptor(dropboxDownloadAction, properties, RestConstants.DROPBOX_CONNECTION_ID);
+
+        final Step dropboxStep = new Step.Builder()
+                .stepKind(StepKind.endpoint)
+                .connection(dropboxConnection)
+                .id(UUID.randomUUID().toString())
+                .action(generateStepAction(dropboxDownloadAction, connectorDescriptor))
+                .configuredProperties(properties)
+                .build();
+
+        steps.getStepDefinitions().add(new StepDefinition(dropboxStep));
+    }
+
+    @And("^create Dropbox upload FINISH action step with file path: \"([^\"]*)\"$")
```

Comment:
Change "And" to "When" (probably).. "And" is just a "shortcut" that defines that this step is the same type as previous one